projects/dashboards/nfdi-gdo/src/app/species-selector/species-selector.component.ts

```diff
@@ -441,7 +441,15 @@ export class SpeciesSelectorComponent implements OnInit, OnDestroy {
         });
         this.subscriptions.push(environmentLayerSubscription);
 
-        this.dataSelectionService.setTimeSteps([...generateYearlyTimeSteps(START_YEAR, END_YEAR, this.currentMonth)]);
+        this.dataSelectionService.setTimeSteps(
+            [...generateYearlyTimeSteps(START_YEAR, END_YEAR, this.currentMonth)],
+            undefined,
+            new Time('2010-01-01T00:00:00.000Z', '2010-02-01T00:00:00.000Z'),
+        );
+
+        this.selectDragonflySpecies('Anax Imperator');
+        this.selectFishSpecies('Abramis brama');
+        this.selectEnvironmentLayer(this.environmentLayers[0]);
```

Comment:
From what I read, there should not be any environmental layer be preselected.

Reply:
b9bbc1bdac6f0091bf57672ed5eb62c06dbd13ec